### DISCUSSION TITLE
Link.tree Feature - Music Scans

### DISCUSSION THREAD

**Discussion by forgeOneHQ** [JulianPrieber/littlelink-custom#745] [Feature requests]
A friend of mine showed me a nice feature which is interesting especially for musicians:
![image](https://github.com/LinkStackOrg/LinkStack/assets/15981620/1abe6018-0a64-4821-abff-cc19d4423112)

When you enter one link he said that link.tree is automatically adding the other links of other platforms.

I have no idea how that works but it is a cool feature.

I am also willing to fund it if there are ideas how that is possile :)

**Comment by MacBadger** [comment]
I'd suggest, make it less platform specific 
I.e. the ability to add the playlist code form any service- Bandcamp, Spotify, Youtube, deezer whatever
